change/@itwin-ec3-widget-react-cf961bfa-2c75-4b8d-adfc-c9ac16da0932.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "added support for localization for ec3 widget",
```

This sentence will appear in changelog. If you will look at the changelog you will see that all entries start with an uppercase letter.

```suggestion
  "comment": "Added support for localization in EC3 widget.",
```

---

done